loops.js

```diff
@@ -17,7 +19,14 @@
 // loop...
 // klaar!
  // ==========================================
-
+const loop = 'loop...';
+for (let i = 0; i < 5; i++) {
+    if (i < 4){
+        console.log(loop);
+    }else{
+        console.log("Klaar!")
```

Comment:
Hier is een **;** vergeten op het einde van regel 27, ik weet dat het niet echt veel uitmaakt XD